index.html

```diff
@@ -25,7 +25,7 @@
   </head>
   <body>
```

Comment:
- [ ] The desktop version of your index page has no footer and partners section currently and it was pointer out by the previous code reviewer. Please consider adding partners and footer section to the index page as it is required to follow the design guidelines. I would advise you to take a look at the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) closely and apply it to your work 😉.

Design guideline:
<img width="668" alt="Loom_0e1EkshtSG" src="https://user-images.githubusercontent.com/81468675/147350640-eb5d82dd-96e4-4764-9361-9b17b7eafdd8.png">